charts/snowplow-rdb-loader-snowflake/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v2
+name: snowplow-rdb-loader-snowflake
+description: A Helm Chart to deploy Snowplow RDB-Loader Snowlake
+version: 0.1.0
+appVersion: "5.0.0"
+icon: https://raw.githubusercontent.com/snowplow-devops/helm-charts/master/docs/logo/snowplow.png
+home: https://github.com/snowplow-devops/helm-charts
+sources:
+  - https://github.com/snowplow-devops/helm-charts
+  - hhttps://github.com/snowplow/snowplow-rdb-loader
+maintainers:
+  - name: jbeemster
+    url: https://github.com/jbeemster
+    email: jbeemster@users.noreply.github.com
+keywords:
+  - snowplow
+  - rdb
+  - rdb-loader
+  - transformer
+  - shredder
+dependencies:
```

Comment:
@Andy-Hay it might be worth reaching out to @jparavisini on these ones - he has recently been through helmifying apps in our internal estate.

My recommendation would be that for the internal deployments you can directly use the generic charts for service-deployment.  If you did want to go ahead and make a specific chart for this application you can also depend directly on the service-deployment chart instead and you just need to setup the values essentially.

You can see an unfinished example of that here: https://github.com/snowplow-devops/helm-charts/pull/45/files

Reply:
Switching over to use the generic charts in service-deployment - should make moving my working Kubernetes provider pods much quicker to convert over to Helm charts.

Will delete this once i've got it progressed.  

Thanks @jbeemster !